tests/functional/oauth_sign_up.js

```diff
@@ -88,6 +90,58 @@ registerSuite('oauth signup', {
         .then(noEmailExpected(email, 1));
     },
 
+    'signup in Chrome for Android, verify same browser': function () {
+      return this.remote
+        .then(openFxaFromRp('signup', {
+          query: {
+            forceUA: 'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Mobile Safari/537.36'
+          }
+        }))
+        .then(testElementTextInclude(selectors.SIGNUP.SUB_HEADER, '123done'))
+        .then(testUrlInclude('client_id='))
+        .then(testUrlInclude('state='))
+
+        .then(fillOutSignUp(email, PASSWORD))
+
+        .then(testElementExists(selectors.CONFIRM_SIGNUP.HEADER))
+        .then(openVerificationLinkInNewTab(email, 0))
+
+        .then(switchToWindow(1))
+        // wait for the verified window in the new tab
+        .then(testElementExists(selectors.SIGNUP_COMPLETE.HEADER))
+        .then(noSuchElement(selectors.SIGNUP_COMPLETE.CONTINUE_BUTTON))
+        // user sees the name of the RP, but cannot redirect
+        .then(testElementTextInclude(selectors.SIGNUP_COMPLETE.SERVICE_NAME, '123done'))
+
+        // switch to the original window
+        .then(closeCurrentWindow())
+
+        .then(testElementExists(selectors.SIGNUP_COMPLETE.HEADER))
+        .then(click(selectors.SIGNUP_COMPLETE.CONTINUE_BUTTON))
+
+        .then(testElementExists(selectors['123DONE'].AUTHENTICATED));
+    },
+
+    'signup in Chrome for Android, verify same tab': function () {
+      return this.remote
+        .then(openFxaFromRp('signup', {
+          query: {
+            forceUA: 'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Mobile Safari/537.36'
+          }
+        }))
+        .then(testElementTextInclude(selectors.SIGNUP.SUB_HEADER, '123done'))
+        .then(testUrlInclude('client_id='))
+        .then(testUrlInclude('state='))
+
+        .then(fillOutSignUp(email, PASSWORD))
+
+        .then(testElementExists(selectors.CONFIRM_SIGNUP.HEADER))
+        .then(openVerificationLinkInSameTab(email, 0))
+
+        // user is immediately redirected
```

Comment:
@vladikoff - What does Chrome for Android do here if the user verifies in the same tab and tries to redirect to a special URL? If they ignore the redirect, we'll have to fix this behavior too. 

Chrome for Android redirects when verifying the email address in the same tab using 123done-dev.lcip.org as the relier.